common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/signup/SignUpContinueApiResponse.kt

```diff
@@ -51,16 +51,15 @@ import java.net.HttpURLConnection
  */
 data class SignUpContinueApiResponse(
     @Expose override var statusCode: Int,
-    @SerializedName("signin_slt") val signInSLT: String?,
-    @SerializedName("signup_token") val signupToken: String?,
+    @SerializedName("continuation_token") val continuationToken: String?,
+    @Expose @SerializedName("expires_in") val expiresIn: Int?,
+    @Expose @SerializedName("error") val error: String?,
+    @Expose @SerializedName("error_codes") val errorCodes: List<Int>?,
+    @Expose @SerializedName("error_description") val errorDescription: String?,
     @Expose @SerializedName("unverified_attributes") val unverifiedAttributes: List<Map<String, String>>?,
```

Comment:
May I ask why @Expose unverified_attributes but invalid_attributes not

Reply:
This is still an open question, right? @GBakolas

Reply:
As discussed the API returns this the invalid_attributes field but it contains only the names of the attributes and no values so we can expose it